waspc/data/Generator/templates/sdk/wasp/client/test/vitest/helpers.tsx

```diff
@@ -31,7 +31,7 @@ export function renderInContext(ui: ReactElement): RenderResult {
   )
   return {
     ...result,
-    rerender: (rerenderUi: ReactElement) =>
+    rerender: (rerenderUi: ReactNode) =>
```

Comment:
@infomiho Thanks for linking the original code.

They indeed use `ReactElement` and then assert the entire object `as any`. [Here](https://github.com/TanStack/query/blob/aadb83122b4dba8926c09e355ffbd994e8e699cf/packages/react-query/src/__tests__/utils.tsx#L18)'s the code.

The docs say:
> Where ReactElement only represents JSX, ReactNode represents everything that can be rendered.

Any idea why they use `ReactElement`. Is that a mistake?

Reply:
I don't have any extra insights other than it might have been an error in `react-query` :)